CHANGES.md

```diff
@@ -3,6 +3,7 @@
 ## Next
 
 * PR #745: Supporting load from query in spark-3.1-bigquery.
+* PR #767: Adding the option readSessionTimeoutInSeconds, to set the max timeout for a read session.
```

Comment:
Nit: I'd like to make it clear that this value specifies an override value for the deadline on `CreateReadSession` rather than an override for the eventual expiration time of the session. The latter is not user-configurable.

Reply:
Okay